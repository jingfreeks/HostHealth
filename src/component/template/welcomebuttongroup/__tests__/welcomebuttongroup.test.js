import React from 'react';
import {render} from '@testing-library/react-native';
import WelcomeButtonGroup from '../welcomebuttongroup';

describe('Welcome Button group Template Component', () => {
  it('Should work as expected to get snapshot', () => {
    const all = render(<WelcomeButtonGroup />);
    expect(all.toJSON()).toMatchSnapshot();
  });
});
