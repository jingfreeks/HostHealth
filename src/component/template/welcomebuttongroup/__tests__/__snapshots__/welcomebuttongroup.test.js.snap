// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome Button group Template Component Should work as expected to get snapshot 1`] = `
<View
  style={
    [
      {
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
      },
    ]
  }
>
  <View
    style={
      [
        {
          "marginLeft": 5,
          "marginRight": 5,
          "paddingBottom": 5,
          "paddingLeft": 5,
          "paddingRight": 5,
          "paddingTop": 5,
        },
      ]
    }
  >
    <View
      border={10}
      borderc="#99E1E1"
      borderw={2}
      style={
        [
          {
            "borderColor": "#99E1E1",
            "borderRadius": 10,
            "borderWidth": 2,
          },
        ]
      }
    >
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={false}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "alignItems": "center",
            "backgroundColor": "#d6f3f3",
            "borderRadius": 10,
            "opacity": 1,
            "paddingBottom": 15,
            "paddingLeft": 15,
            "paddingRight": 15,
            "paddingTop": 15,
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "#03606A",
                "fontFamily": "Avenir",
                "fontSize": 15,
                "fontWeight": "500",
              },
            ]
          }
        >
          SIGN UP
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      [
        {
          "marginLeft": 5,
          "marginRight": 5,
          "paddingBottom": 5,
          "paddingLeft": 5,
          "paddingRight": 5,
          "paddingTop": 5,
        },
      ]
    }
  >
    <View
      border={10}
      borderc="#99E1E1"
      borderw={2}
      style={
        [
          {
            "borderColor": "#99E1E1",
            "borderRadius": 10,
            "borderWidth": 2,
          },
        ]
      }
    >
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={false}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "alignItems": "center",
            "backgroundColor": "#FFFFFF",
            "borderRadius": 10,
            "opacity": 1,
            "paddingBottom": 15,
            "paddingLeft": 15,
            "paddingRight": 15,
            "paddingTop": 15,
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "#03606A",
                "fontFamily": "Avenir",
                "fontSize": 15,
                "fontWeight": "500",
              },
            ]
          }
        >
          SIGN IN
        </Text>
      </View>
    </View>
  </View>
</View>
`;
